guides/rif-relay/integrate.md

```diff
@@ -40,108 +43,254 @@ For more details on this, please refer to the [Architecture page](/rif/relay/arc
+  });
+
+  setProvider(ethersProvider);
+```
 
-    const contractAddresses: RelayingServicesAddresses = {
-        relayHub: '0xE0825f57Dd05Ef62FF731c27222A86E104CC4Cad',
-        smartWallet: '0x73ec81da0C72DD112e06c09A6ec03B5544d26F05',
-        smartWalletFactory: '0x03F23ae1917722d5A27a2Ea0Bcc98725a2a2a49a',
-        smartWalletDeployVerifier: '0x1eD614cd3443EFd9c70F04b6d777aed947A4b0c4',
-        smartWalletRelayVerifier: '0x5159345aaB821172e795d56274D0f5FDFdC6aBD9',
-        // TODO: Why aren't these addresses required? we may set them as optional
-        penalizer: '',
-        customSmartWallet: '',
-        customSmartWalletFactory: '',
-        customSmartWalletDeployVerifier: '',
-        customSmartWalletRelayVerifier: '',
-        sampleRecipient: ''
-    };
+Where variables are:
+
+  * **CHAIN_ID**: The chain id that identify the network that the client will be interacting.
+  * **SERVER_URL_ARRAY**: An array of relay server url's that the client can interact with.
+  * **RELAY_HUB_ADDRESS**: The relay hub contract address.
+  * **DEPLOY_VERIFIER_ADDRESS**: The deploy verifier contract address.
+  * **RELAY_VERIFIER_ADDRESS**: The relay verifier contract address.
+  * **SMART_WALLET_FACTORY_ADDRESS**: The smart wallet factory contract address.
+
+After setting the configuration and the ethers provider, we can create start creating instances from the `Relay Client`.
+
+### Account Manager
+
+The `Account Manager` manager its a singleton component from the RIF Relay Client library that helps to sign the relay transactions.  This component can sign the transactions with an internal account that was previously added or using a wallet provider like [metamask](https://metamask.io/). The `Account Manager` will look first for internal account's and if none is found, will try to use the provider that was [previously setup](/guides/rif-relay/integrate/#relay-client).
+
+The `Account Manager` accepts [Ethers V5 Wallets](https://docs.ethers.org/v5/api/signer/#Wallet) as internal accounts.  
+
+To interact with the `Account Manager` we need to the next steps:
 
-    const web3 = new Web3(<RSK_NODE_ENDPOINT>);
+1. Get an instance.
+2. Add a new account.
 
-    const relayingServices = new DefaultRelayingServices(web3);
-    await relayingServices.initialize(config, contractAddresses, {
-        loglevel: 1
-    });
+```typescript
+    import {
+      AccountManager,
+    } from '@rsksmart/rif-relay-client';
+
+    const accountManager = AccountManager.getInstance();
+
+    accountManager.addAccount(<INTERNAL_ACCOUNT_OBJECT>);
 ```
-> All the contracts addresses and the chainId are from RSK Node Regtest
 
 Where variables are:
 
-  * **RSK_CHAIN_ID**: The RSK node chain ID which is running (e.g. `33`).
-  * **RIF_RELAY_SERVER_URL**: The RIF Relay Server URL (e.g. `http://localhost:8090`).
-  * **RSK_NODE_ENDPOINT**: The RSK node endpoint which is running (e.g. `http://localhost:4444`).
+  * **INTERNAL_ACCOUNT_OBJECT**: Ethers V4 Wallet object.
```

Comment:
Line 93 states that the Account Manager accepts V5 wallets.

Reply:
updated